src/main/java/frc/robot/commands/ControlPanelColorCommand.java

```diff
@@ -1,41 +1,39 @@
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.common.ControlPanelColorSensor.ControlPanelColor;
 import frc.robot.subsystems.ControlPanelSubsystem;
 
 public class ControlPanelColorCommand extends CommandBase {
-    private ControlPanelSubsystem controlPanel;
-    private ControlPanelColor turnToColor;
+  private ControlPanelSubsystem controlPanel;
+  private float direction;
 
-    public ControlPanelColorCommand(ControlPanelColor turnToColor) {
+    public ControlPanelColorCommand() {
         // TODO: Why won't this line do what we want? Let's think about it
         controlPanel = new ControlPanelSubsystem();
-        this.turnToColor = turnToColor;
     }
 
-    @Override
-    public void initialize() {
-        // Extend the solenoid if it isn't already
-        controlPanel.setSolenoid(true);
-    }
+  @Override
+  public void initialize() {
+    controlPanel.setSolenoid(true);
+    direction = Math.signum(controlPanel.getTurnToColor()
+        .ordinal() - controlPanel.getDetectedColor()
+        .ordinal());
+  }
 
-    @Override
-    public void end(boolean interrupted) {
-        controlPanel.setSolenoid(false);
-        controlPanel.spin(0);
-    }
+  @Override
+  public void end(boolean interrupted) {
+    controlPanel.setSolenoid(false);
+    controlPanel.spin(0);
+  }
 
-    @Override
-    public boolean isFinished() {
-        // TODO: Implement equals for ControlPanelColor and use it here
-        return controlPanel.getControlPanelColor().getColor().equals(turnToColor.getColor());
-    }
+  @Override
+  public boolean isFinished() {
+    return controlPanel.getDetectedColor()
+        .equals(controlPanel.getTurnToColor());
```

Comment:
Enums already support `::equals`

Reply:
True, I'm not sure why I had it compare the color values. In fact, I believe you can just use `==` with enums too.